app/assets/javascripts/templates/partials/shop_variant_no_group_buy.html.haml

```diff
@@ -8,7 +8,7 @@
     {{ "js.shop_variant_no_group_buy.increment" | t }}
   %br
   .variant-quantity-display{ng: {class: "{visible: variant.line_item.quantity}"}}
-    {{ "js.shop_variant_no_group_buy.in_cart" | t:{quantity: variant.line_item.quantity} }}
+    {{ "js.shop_variant_no_group_buy.in_cart" | t:{quantity: variant.line_item.quantity || 0} }}
```

Comment:
same here, some context would be better I think.
js.shopfront.shop_variant_no_group_buy

Reply:
Yeah, I find that tricky. I tried to follow the filename of the template but

- all js templates are in one directory and
- this key is actually used in two templates.

I think I will name it `js.shopfront.variant.in_cart` and `js.shopfront.variant.add_to_cart`.